pkgs/development/interpreters/lua-5/interpreter.nix

```diff
@@ -40,14 +40,17 @@ self = stdenv.mkDerivation rec {
     "PLAT=macosx"
   ] else [
     "PLAT=linux"
-  ])
+  ]) ++ (if stdenv.buildPlatform != stdenv.hostPlatform then [
```

Comment:
do  we need the if there ? if platfroms are equal then it won't matter

Reply:
I think it's cleaner that way. Because if platforms are equal, we don't even need these args, we'll let the build system sort it out.

Reply:
@teto Additionally, this shouldn't cause a rebuild for people who are not cross-compiling!

Reply:
I am not that familiar with cross compilation. Would like the opinon of @7c6f434c too

Reply:
Looks reasonable